website/python/run_server.py

```diff
@@ -1,8 +1,8 @@
 import os
 
-os.environ['BREWMONITOR_CONFIG'] = './debug_config.yaml'
-
 from brewmonitor.app import make_app
 
+os.environ['BREWMONITOR_CONFIG'] = './debug_config.yaml'
```

Comment:
Same thing as before, I think we should have an if not def, then set to that default value.

Reply:
This is explicitly to run as debug while developing, as you can see the secret in the line below is very secret... But I added a comment to make sure it's clear.